source/features/make-discussion-list-toolbar-sticky.css

```diff
@@ -0,0 +1,7 @@
+/* Sticky table header on issues list */
+.rgh-sticky-discussion-list-toolbar #js-issues-toolbar {
+	position: sticky;
+	top: 0;
+	/* Must be above .modal-backdrop (z-index 20) #1317 */
+	z-index: 25;
+}
```

Comment:
I'd just keep the CSS instead. This feature is minor enough and does not have the logic that the sidebar one has